### ISSUE TITLE
no such file or directory, copyfile '.forceignore' -> 'test/.forceignore'

### ISSUE THREAD

**Issue by sfdcale** [Accenture/sfpowerkit#104]
```
> sfdx sfpowerkit:project:diff -d test -t HEAD -r HEAD~1
ERROR running sfpowerkit:project:diff:  ENOENT: no such file or directory, copyfile '.forceignore' -> 'test/.forceignore'
```

Receiving the above error.

**Comment by sfdcale** [comment]
workaround that is working for now is : 
```
> touch .forceignore
> sfdx sfpowerkit:project:diff -d test -t HEAD -r HEAD~1
```

**Comment by azlam-abdulsalam** [comment]
@sfdcale  We have fixed this in our upcoming 1.20.X release and will hopefully hit the stable channel tomorrow. Thanks again !
